packages/fxa-auth-server/lib/senders/emails/partials/location/mocks.ts

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const MOCK_LOCATION = {
```

Comment:
It'd probably benefit us to do this for some other variables too in future PRs, like `passwordChangeLink` / anything else we're having to copy and paste between stories.